src/test/java/com/ifactory/press/db/solr/spelling/suggest/MultiSuggesterTest.java

```diff
@@ -120,8 +135,13 @@ private Suggestion assertSuggestionCount(String prefix, int count, String sugges
             Logger.getLogger(MultiSuggesterTest.class.getName()).log(Level.SEVERE, null, ex);
         }
         SpellCheckResponse scr = resp.getSpellCheckResponse();
-        assertNotNull("no spell check reponse found", scr); // null happens for empty assertions test
-        Suggestion suggestion = scr.getSuggestion(prefix);
+        
+        Suggestion suggestion = null;
+        // Assumption: Either null is a case for no results or a bug in Solr after 5.0

```

Comment:
This should only be the case for single char results. Because of this bug https://issues.apache.org/jira/browse/SOLR-11732